AzureDevops.Authentication/Src/AadAuthentication.cs

```diff
@@ -70,22 +70,22 @@ public sealed class AadAuthentication : Authentication, IAadAuthentication
         internal AadAuthentication(
             RuntimeContext context,
             ICredentialStore personalAccessTokenStore,
-            ITokenStore vstsIdeTokenCache,
-            IAuthority vstsAuthority)
+            ITokenStore vsIdeTokenCache,
+            IAuthority devopsAuthority)
```

Comment:
azureDevOpsAuthority? Or is that too lengthy?

Reply:
kind of lengthy... and rather difficult to read in code.